.github/workflows/s3-deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to S3
+on:
+  push:
+    branches:
+      - storybook6
+    paths-ignore:
+      - package.json
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - name: Install dependencies
+        run: yarn install
+      - name: Build storybook
+        run: yarn storybook:build
+      - name: Upload storybook to S3
+        uses: jakejarvis/s3-sync-action@master
```

Comment:
Might wanna lock this version and not use `master`.

Reply:
Looks like there is no a fixed version tag can be used from their document. 🤔